express-delete/app.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const app = express();
+const PORT = 3000;
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res, next) => {
+  res.json(gradeArray(grades));
+});
+
+app.delete('/api/grades/:id', (req, res, next) => {
+  delete grades[req.params.id];
+  res.json(gradeArray(grades));
```

Comment:
Here you only needed to send back a status code of `204` using `res.sendStatus(204)`